<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelDialog</class>
 <widget class="QDialog" name="ControlPanelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanelDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>SceneType</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_nonote">
        <property name="text">
         <string>NoNote</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_powerflow">
        <property name="text">
         <string>PowerFlow</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_name">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_resistance">
        <property name="text">
         <string>Resistance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_type">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>21</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>91</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Shown</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBox_stationname">
        <property name="text">
         <string>Station Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_branchline">
        <property name="text">
         <string>Branch Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_reactivepowerflow">
        <property name="text">
         <string>ReactivePower Flow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_angle">
        <property name="text">
         <string>Voltage Angle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_stationvalue">
        <property name="text">
         <string>Station Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_branchvalue">
        <property name="text">
         <string>Branch Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_admittance">
        <property name="text">
         <string>Admittance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_custom">
        <property name="text">
         <string>All Node Voltage/custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>201</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Unit</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_unit1">
      <property name="text">
       <string>Actual Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_unit2">
      <property name="text">
       <string>Per_Unit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
